keras/utils/conv_utils.py

```diff
@@ -49,7 +49,7 @@ def convert_data_format(data_format, ndim):
         'Expected values are ["channels_first", "channels_last"]')
 
 
-def normalize_tuple(value, n, name):
+def normalize_tuple(value, n, name, cmp=None):
```

Comment:
Please change the cmp to validation_fn, and the function should return False for illegal inputs.

The lambda function above should also be changed to lambda x: x > 0 instead, which is more readable (not using negation).

Reply:
Done. What's more, I add an alternative text for validation fn. If it looks fine, please resolve this conversation. @qlzh727

Reply:
The lambda function is kept here because normalize_tuple function has checked the illegal inputs. It is unnecessary to do it again in validation_fn.

Reply:
Now both the method signature and the caller are quite weird now. 

Seems that there are only true use case, and majority of them are strict positive. I don't think there will be any new use case, eg different lambda. So let's just add a kwarg for allow_zero and default to False, which will allow the caller side to skip all the labmdas and msgs.

Reply:
Done